scatterplot.Rmd

```diff
@@ -140,6 +140,11 @@ scatter +
 `ggplot2` makes it very easy to create scatterplots. Using `geom_point()`, you can easily plot two different aesthetics in one graph. It also is simple to add on extra formatting to make your plots look nice (All that is really necessary is the data, the aesthetics, and the geom).
 
 ## Theory
+
+Scatterplots are very useful in understanding the correlation (or lack thereof) between variables.  For example, in section 13.1 notice the positive relationship between brain and body weight in species of land mammals. The scatterplot gives a good idea of whether that relationship is positive or negative and if there is correlation.  Don't mistake correlation in a scatterplot for causation! 
```

Comment:
```suggestion
Scatterplots are very useful in understanding the correlation (or lack thereof) between variables.  For example, in [section 13.2](scatter.html#tldr-7) notice the positive relationship between brain and body weight in species of land mammals. The scatterplot gives a good idea of whether that relationship is positive or negative and if there's a correlation.  However, don't mistake correlation in a scatterplot for causation! 
```

Reply:
Please update with these small changes. Make sure I did the link correctly :)

Reply:
I think it worked.  As for the differentiation between "when to use" and "theory", I think the "when to use" can be a bit more to the point, while the theory gives the best practices and how to decide between variations in the graph.